src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialWashPlant.java

```diff
@@ -138,8 +140,7 @@ public boolean checkMachine(IGregTechTileEntity aBaseMetaTileEntity, ItemStack a
 
     @Override
     protected IAlignmentLimits getInitialAlignmentLimits() {
-        // don't rotate a washer, water will flow out.
```

Comment:
this is exactly why it was not made rotatable in the first place.

Reply:
You can block off the "top" to stop the water flowing out. Besides that, letting the water flow out is also kind of cool looking, if you build a catchment for it.

Reply:
if you need extra handling when the multiblock is rotated, that's means it shouldn't being rotated.

It makes no sense that the washer should be rotated, as there is nothing happening in the water, you see nothing.

Reply:
The extra handling to fill up the washer is there regardless of this change.

Reply:
did you know the washer is filling itself up?

Reply:
Yes, of course. That's what the changes in checkForWater are for, they make sure the multi fills itself up correctly even when rotated